src/history/HistoryArchive.h

```diff
@@ -96,16 +98,29 @@ struct HistoryArchiveState
     void
     serialize(Archive& ar)
     {
-        ar(CEREAL_NVP(version), CEREAL_NVP(server), CEREAL_NVP(currentLedger),
-           CEREAL_NVP(currentBuckets));
+        ar(CEREAL_NVP(version), CEREAL_NVP(server), CEREAL_NVP(currentLedger));
+        try
+        {
+            ar(CEREAL_NVP(networkPassphrase));
+        }
+        catch (cereal::Exception)
+        {
+            // networkPassphrase wasn't parsed.
+            // This is expected when the input file does not contain it.
+        }
+        ar(CEREAL_NVP(currentBuckets));
     }
 
     template <class Archive>
     void
     serialize(Archive& ar) const
     {
-        ar(CEREAL_NVP(version), CEREAL_NVP(server), CEREAL_NVP(currentLedger),
-           CEREAL_NVP(currentBuckets));
+        ar(CEREAL_NVP(version), CEREAL_NVP(server), CEREAL_NVP(currentLedger));
+        if (networkPassphrase != "")
```

Comment:
the proper way to check this is to use `.empty()`

Reply:
Done!